src/main/java/de/tum/in/www1/artemis/repository/StudentParticipationRepository.java

```diff
@@ -207,7 +293,7 @@ and not exists (select prs from participation.results prs
     Optional<StudentParticipation> findWithEagerResultsAndFeedbackById(@Param("participationId") Long participationId);
 
     /**
-     * Find the participation with the given id. Additionally, load all the submissions and results of the participation from the database. Returns an empty Optional if the
+     * Find the participation with the given id. Additionally, load all the submissions and results of the participation FROM the database. Returns an empty Optional if the
```

Comment:
Stray string replace. 😅

Reply:
replaced them, I guess I accidentally changed  them when I was switching everything to uppercase